docs/whitepaper.md

```diff
@@ -14,17 +14,17 @@ The broad adoption of computers in scientific research has made it possible to s
+Often, data must undergo the painful process of "munging," where the heterogeneity of various sources are integrated together to form source dataset(s). Although data collection and storage across all scientific disciplines has increased substantially, reproducibility of computational analysis still remains primitive. For example, most studies depend on data sources that are stored in a variety of incompatible formats. Researchers persistently transform this data for analyses, building an (often collaborative) workflow to ensure interpretation of accurate data. However, it is often difficult to keep track of forks in the datastream, resulting in increasing interconnectedness and complexity of source data. In other words, analyses that result in significant, irreconcilable transformations increase data heterogeneity, and thus headache for a team.
 
 A sustainable and scalable collaborative workflow, also known as a data pipeline, is built to ensure correctness during this kind of collaborative work. Unfortunately, building this infrastructure is difficult, expensive, and time-consuming, especially for domain experts. This means that data-intensive research is often limited to those with the necessary computational skills, personal and professional connections, or institutional resources to build, maintain, and ensure reproducibility of a complex data pipelining system. This *data divide* explicitly "demarcates research 'insiders' and 'outsiders'... undermin[ing] the research community" [1]. If only the original authors are able to reproduce scientific discoveries, the scientific process is broken.
 
-To close the data divide, we must build tools that allow researchers to be able to focus more on scientific discovery and less on integrating disparate data. We introduce Dat, a version-controlled distributed database and data tool that has user interface of a version control system (VCS). Key features of dat include the ability to *track changes* in tabular and binary formats; *supply access points* across peer-to-peer internal and public networks; *create historical checkpoints* with metadata; *replicate* quickly at a given checkpoint; and *encourage forking* of data streams.
+To close the data divide, we must build tools that allow researchers to be able to focus more on scientific discovery and less on integrating disparate data. We introduce Dat, a version-controlled distributed database and data tool that has the user interface of a version control system (VCS). Key features of dat include the ability to *track changes* in tabular and binary formats; *supply access points* across peer-to-peer internal and public networks; *create historical checkpoints* with metadata; *replicate* quickly at a given checkpoint; and *encourage forking* of data streams.
```

Comment:
> We introduce Dat, a version-controlled distributed database and data tool that has the user interface of a version control system (VCS).

This seems longer than it needs to be. What about "Distributed version control system for data"? I recognize this is more complete, but this is almost the same definition of git.